src/lib/fast_check.py

```diff
@@ -69,7 +83,7 @@ async def get_dynatrace_token_metadata(dt_session: ClientSession, context: Loggi
         return {}
 
 
-class FastCheck:
+class MetricsFastCheck:
```

Comment:
Referring to our discussion about metrics fast check and disabled monitoring due to missing permissions (when an annotation is not created for the service account) - I checked and after your changes pod will be restarted until annotation is created, so after that monitoring will start working properly.

But I think we could change one log : from "Access to following projects: " to e.g. "There is no access to any projects. Check service account configuration." when 'projects' is empty. Actually I added this log, so I can change it in my PR.
What do you think? Would it be more readable for customer?

Reply:
Thank you for checking this scenario. Updating log message is neglible change, so I think that we can include it in your PR.